Improve reliability for 6GB+ RAM devices

dhinakg opened the pull request:
- Drop puaf pages to 256 on all devices except A8/A9
- Hog up to 4GB of memory on all devices (this makes 16GB+ work now)
- Bump grabbed_free_pages_max to 200000 on 6GB+ RAM devices
- Bump grabbed_free_pages_max to 800000 on 16GB+ RAM devices

<details>
<summary>Old changes</summary>

- Use 3072 puaf pages on 6GB+ RAM devices
- Hog up to 4GB of memory on 16GB+ RAM devices (this makes 16GB+ work now)
- Bump grabbed_free_pages_max to 800000 on 16GB+ RAM devices
</details>

dhinakg committed:
Improve reliability for 6GB+ RAM devices

- Use 3072 puaf pages on 6GB+ RAM devices
- Hog up to 4GB of memory on 16GB+ RAM devices
- Bump grabbed_free_pages_max to 800000 on 16GB+ RAM devices

dhinakg committed:
Add newline

corruptedkiwi commented:
Can confirm fixed ip14 pro

corruptedkiwi commented:
Confirmed ip13 pro as well

dhinakg committed:
Merge remote-tracking branch 'public/2.x' into 2.0-16gb

awareunlikeu commented:
Is there anyway for 2.0 to support arm64e devices on 16.6?  I’m on 8th gen iPad running 16.6

dhinakg committed:
Adjust kfd parameters

- Drop puaf_pages to 256 for all devices (excluding A8/A9)
- Enable memory hogging on all devices
- Bump grabbed_free_pages_max to 200000 on 6GB+ RAm devices
